msg/GPSPosition.msg

```diff
@@ -0,0 +1,14 @@
+builtin_interfaces/Time time_stamp # Timestamp from when the message was send
+string sender_id # Name of the node that send this command
+float64 latitude_deg # Latitude in degrees (range: -90 to +90)
+float64 longitude_deg # Longitude in degrees (range: -180 to +180)
+float64 relative_altitude_m # Altitude relative to takeoff altitude in metres
+uint8 fix_type # GPS Fix type
+#enum
+uint8 NO_GPS = 0 # No GPS connected
+uint8 NO_FIX = 1 # No position information, GPS is connected
+uint8 FIX_2D = 2 # 2D position
+uint8 FIX_3D = 3 # 3D position
```

Comment:
Please clarify in the comment that this means that a 2D / 3D information is available (i.e. a status) rather than those variables being the values for 2D / 3D values themselves. For example "Position is 2D" would be more a more understandable comment in this line than only prefixing the whole paragraph "#enum".

Reply:
Fixed: https://github.com/THI-Drone/interfaces/pull/3/commits/3a7b4f248c2a4cf34bb4702476930c6d5eaf1023

Reply:
Sorry, copied pasted by accident. Indeed, this is still an open topic.

Reply:
Fixed this time: https://github.com/THI-Drone/interfaces/pull/3/commits/1bb5977f541a861f04628d7b66853d790381579c

Reply:
thanks